app/router.js

```diff
@@ -1,127 +1,45 @@
-      payload.issue.node_id,
-      reason
-    );
-    return;
-  }
-
-  const reopenMatches = reopenMatcher(payload.comment.body);
-  if (reopenMatches) {
-    console.log(
-      `${id} Re-opening issue ${payload.issue.html_url} ${actorRequest}`
-    );
-    await reopenIssue(
-      await getAuthToken(auth, payload.installation.id),
-      sourceRepo,
-      payload.issue.node_id
-    );
-    return;
-  }
+import deepmerge from "deepmerge";
+import { getCommands, noneMatch } from "./commands.js";
 
-  const labelMatches = labelMatcher(payload.comment.body);
-  if (labelMatches) {
-    const labels = extractCommaSeparated(labelMatches[1]);
-
-    console.log(
-      `${id} Labeling issue ${payload.issue.html_url} with labels ${labels} ${actorRequest}`
-    );
-    await addLabel(
-      await getAuthToken(auth, payload.installation.id),
-      payload.repository.owner.login,
-      sourceRepo,
-      payload.issue.node_id,
-      labels
-    );
-    return;
-  }
-
-  const removeLabelMatches = removeLabelMatcher(payload.comment.body);
-  if (removeLabelMatches) {
-    const labels = extractCommaSeparated(removeLabelMatches[1]);
+export async function router(auth, id, payload, verbose) {
+  const sourceRepo = payload.repository.name;
 
-    console.log(
-      `${id} Removing label(s) from issue ${payload.issue.html_url}, labels ${labels} ${actorRequest}`
-    );
-    await removeLabel(
-      await getAuthToken(auth, payload.installation.id),
-      payload.repository.owner.login,
-      sourceRepo,
-      payload.issue.node_id,
-      labels
-    );
-    return;
-  }
+  const commands = getCommands(id, payload);
 
-  const reviewerMatches = reviewerMatcher(payload.comment.body);
-  if (reviewerMatches) {
-    console.log(
-      `${id} Requesting review for ${reviewerMatches[1]} at ${payload.issue.html_url} ${actorRequest}`
-    );
-    const reviewers = extractUsersAndTeams(
-      payload.repository.owner.login,
-      reviewerMatches[1]
-    );
-    await requestReviewers(
-      await getAuthToken(auth, payload.installation.id),
-      payload.repository.owner.login,
-      sourceRepo,
-      payload.issue.node_id,
-      reviewers.users,
-      reviewers.teams
-    );
+  if (noneMatch(commands)) {
+    if (verbose) {
+      console.log("No match for", payload.comment.body);
+    }
     return;
   }
 
-  if (verbose) {
-    console.log("No match for", payload.comment.body);
+  const authToken = await getAuthToken(auth, payload.installation.id);
+  const OctokitConfig = Octokit.plugin(octoKitConfig);
+  const octokit = new OctokitConfig({ auth: authToken });
+
+  // TODO validate against schema
+  // noinspection JSUnusedGlobalSymbols
+  const { config } = await octokit.config.get({
+    owner: payload.repository.owner.login,
+    repo: sourceRepo,
+    path: ".github/comment-ops.yml",
+    defaults: (configs) => deepmerge.all([defaultConfig, ...configs]),
+  });
+
+  const runCommands = Object.values(commands).filter((command) =>
```

Comment:
I should have provided this in a single review, my bad.
```suggestion
  const runCommands = commands.filter((command) =>
```

Reply:
it's a few changes needed, the filter has stopped working, debugging it atm

Reply:
that works now, just some tests to look at after some refactoring changed the args